src/js/wizards/Sell/5_Margin/components/MarginSelectorForm.jsx

```diff
@@ -54,10 +55,12 @@ class MarginSelectorForm extends Component {
         </div>
         <small>{t('marginSelectorForm.priceOrigin')}</small>
 
-        <h3>{t('marginSelectorForm.ourFee')}</h3>
-        <div className="border rounded p-3">
-          TODO SNT
-        </div>
+        {fee !== '0' && <Fragment>
```

Comment:
From the container, it looks like while we are fetching the fee, it would be undefined while not resolved yet. 
Maybe here we should check that the fee is not undefined too?